### ISSUE TITLE
(Feature Request) Minimalist Mode: Transparent Background, Smaller Icons, Tiny Progress Bar with Tiny Text [Friendly for small Monitors]

### ISSUE THREAD

**Issue by Kristijan1001** [joshhunt/GhostOverlay#1]
Maybe we can have a Minimalist Mode (dunno if it is the right terminology for it) down the road: Transparent Background, Smaller Icons, Tiny Progress Bar with Tiny Text which will be friendly with small monitors.

**Comment by boomboompower** [comment]
Agreed, the ability to change background opacity would be amazing, maybe even some sort of scaling/resizing mode as well?

Additionally, maybe toggling components such as the description or character name will be useful as well.

**Comment by joshhunt** [comment]
Yup, agree fully. 

 - Transparency will come mid next month, once Xbox Game Bar supports it
 - I will make the widget more responsive, so when you resize it to be smaller, things will shrink down a bit more
 - I'll add a setting to auto hide complete bounties, and maybe even bounty descriptions

**Comment by hearnia2k** [comment]
Another feature that would be awesome would be multiple columns.  I have an ultrawide display, and it would be cool to stretch the overlay across a large width, with maybe 3-4 columns,

Also, ability to launch outside of game bar should be possible, from what I read on the SDK, and this could be super useful for running it on a second display

**Comment by hearnia2k** [comment]
Adding another related suggestion here:  Can we have a selection of (vertical) text alignment?  I'd prefer that things were aligned to the bottom of the window, rather than the top.  This means I can have a taller window, and all my tracked bounties are stacked from the bottom, and as they get completed and are no longer tracked I don't need to resize the window every time.

**Comment by joshhunt** [comment]
Would a mode that automatically removed completed bounties and shrink the
window solve the same problem? I’ve been thinking of this for a while.

**Comment by hearnia2k** [comment]
It depends on how it shrinks the window, in terms of window alignment.  I prefer GhostOverlay in the bottom right corner.  Most likely resizing the window would retain the top left corner as the fixed position, however, in my case I'd want the bottom right to be my fixed position, effectively lowering the top.  

Overall, if the window size reduced, and I could pick my fixed corner it would be a better solution, since this would also mean that the window background area is reduced to the minimum required for the number of tracked bounties.

Hopefully this makes sense!
